migrations/20160422160813_create_story.js

```diff
@@ -9,7 +9,9 @@ exports.up = function(knex, Promise) {
 		t.string('title').notNull();
 		t.string('author').nullable();
 		t.string('coverImage').nullable();
+		t.text('description').nullable();
 		t.enu('category', ['fable']).nullable();
+
```

Comment:
You shouldn't have to modify this migration for this ticket. Undo your changes to this file please